test/parallel/test-http-client-aborted-event.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+require('../common');
+var http = require('http');
+var assert = require('assert');
```

Comment:
It should be perfectly fine to change these just here – formatting changes tend to mess up the history/`git blame`, so it’s always easier to do them in new code/code that’s being touched anyway.